From a9a09f5114c60e9900bc3a5663486d42ff5c2e73 Mon Sep 17 00:00:00 2001
From: Alan Modra <amodra@gmail.com>
Date: Wed, 10 Nov 2021 10:26:10 +1030
Subject: [PATCH 81/97] Re: Add --unicode option

At low optimisation levels gcc may warn.

	* strings.c (print_unicode_stream_body): Avoid bogus "may be
	used unitialised" warning.
---
 binutils/strings.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/binutils/strings.c b/binutils/strings.c
index f50badf983e..e8649a80d6a 100644
--- a/binutils/strings.c
+++ b/binutils/strings.c
@@ -924,7 +924,7 @@ print_unicode_stream_body (const char *     filename,
   uint num_read = 0;
   uint num_chars = 0;
   uint num_print = 0;
-  int c;
+  int c = 0;
 
   /* Find a series of string_min characters.  Put them into print_buf.  */
   do
-- 
2.34.0

